### PR TITLE
[sql/en] Adding SQL entry

### PR DESCRIPTION
**Pull request by bobdc**
- [x] I solemnly swear that this is all original content of which I am the original author (some of it copied from a blog entry I wrote; all content is mine)
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
Closes #400.

File: sql.html.markdown [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Implementations typically provide a command line prompt where you can enter the commands shown here interactively, and they also offer a way to execute a series of these commands stored in a script file.  (Showing that you’re done with the interactive prompt is a good example of something that isn’t standardized--most SQL implementations support the keywords QUIT, EXIT, or both.)
+
+Several of these sample commands assume that the [MySQL employee sample database](https://dev.mysql.com/doc/employee/en/) available on [github](https://github.com/datacharmer/test_db) has already been loaded. The github files are scripts of commands, similar to the relevant commands below, that create and populate tables of data about a fictional company’s employees. The syntax for running these scripts will depend on the SQL implementation you are using. A utility that you run from the operating system prompt is typical. 
+
+
+```
```
Comment (divayprakash) [inline sql.html.markdown]: Please correct this to 

```
```sql
```

for code highlighting to work correctly.

File: sql.html.markdown [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+language: SQL
+filename: learnsql.sql
+contributors:
+- ["Bob DuCharme", "http://bobdc.com/"]
+lang: en-en
```
Comment (divayprakash) [inline sql.html.markdown]: Please remove this line

```
lang: en-en
```

This is not necessary for English language docs and creates an issue where they are not added to the site build.

File: sql.html.markdown [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---
+language: SQL
+filename: learnsql.sql
+contributors:
+- ["Bob DuCharme", "http://bobdc.com/"]
```
Comment (divayprakash) [inline sql.html.markdown]: Please indent this line using 2 spaces ie.

```
contributors:
  - ["Bob DuCharme", "http://bobdc.com/"]
```

File: sql.html.markdown (hunk 1 above)
Comment (bobdc) [reply sql.html.markdown]: I had tried that with a draft that I keep on bitbucket and it didn't look very good. It looks great on git!

File: sql.html.markdown (hunk 2 above)
Comment (bobdc) [reply sql.html.markdown]: I made the change and resolved it, but I think that it broke the build according to https://travis-ci.org/adambard/learnxinyminutes-docs/builds/448955911?utm_source=github_status&utm_medium=notification .

File: sql.html.markdown [hunk 4]
```diff
@@ -0,0 +1,106 @@
+---
+language: SQL
+filename: learnsql.sql
+contributors:
+  - ["Bob DuCharme", "http://bobdc.com/"]
+https://github.com/adambard/learnxinyminutes-docs
```
Comment (divayprakash) [inline sql.html.markdown]: @bobdc This line with the link to this repo is breaking the build. Please remove this.

Comment (divayprakash) [reply sql.html.markdown]: Line 6

Comment (bobdc) [reply sql.html.markdown]: Sorry, that must have been an accidental paste. Removed.

**Comment by divayprakash** [comment]
@bobdc LGTM!

**Pull request merged**